sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -199,7 +187,7 @@ object DataSourceAnalysis extends Rule[LogicalPlan] with CastSupport {
 
       // Sanity check
       if (t.location.rootPaths.size != 1) {
-        throw new AnalysisException("Can only write data to relations with a single path.")
```

Comment:
The original error message is `Can only write data to relations with a single path`.

We should rename `cannotWriteDataToRelationWithSinglePathError` to `canOnlyWriteDataToRelationWithSinglePathError `

Reply:
OK